extensions/extensions-helm/aissemble-spark-operator-chart/templates/rbac.yaml

```diff
@@ -38,6 +38,7 @@ rules:
   - delete
   - update
   - list
+  - patch
```

Comment:
i: Required to have path action for services

Reply:
Q/S: Did we ever figure out why this is an issue after the upgrade but wasn't before?  I don't see why Spark Operator would suddenly be using a new permission that it hadn't previously.

Reply:
Our custom rbac was not updated to match the spark-operator chart when that was updated to 1.4.6.